### PR TITLE
Feature/update link styles

### PR DESCRIPTION
**Pull request by hdfedscottquested**

### CODE REVIEW

File: components/atoms/link/link.scss [hunk 1]
```diff
@@ -1,46 +1,90 @@
...
 		}
 
-		&:active,
-		&:focus {
-			color: lighten($color-brand-secondary, 10%);
+		&-on-dark {
+			color: $color-white;
+
+			&:hover {
+				color: $color-white;
```
Comment (hdfeddanjackson) [inline components/atoms/link/link.scss:66]: You could remove the `color` properties from line 66 and 76 as this is already being set in the parent selector.

File: components/atoms/link/link.scss [hunk 2]
```diff
@@ -1,46 +1,90 @@
...
-	}
+			color: $color-brand-blue-dark;
 
-	&--secondary {
-		color: $color-brand-secondary;
-		
-		&:hover {
-			color: lighten($color-brand-secondary, 5%);
-			text-decoration: none;
+			&:before {
```
Comment (hdfeddanjackson) [inline components/atoms/link/link.scss:51]: Where you have the pseudo code that repeats, you could reduce the number of code lines by using:

```scss
&:before,
&:after {
	width: 100%;
	background-color: $color-brand-blue-grey-lighter;
}
```

File: components/atoms/link/link.scss [hunk 3]
```diff
@@ -1,46 +1,90 @@
 @import '../../../src/scss/variables/variables';
+@import '../../../src/scss/global/type';
 
 .a-link {
-	color: $color-dark;
-	text-decoration: underline;
+	position: relative;
+	z-index: 1;
+	text-decoration: none;
```
Comment (hdfeddanjackson) [inline components/atoms/link/link.scss:7]: I know this is a design decision and not something we can change, but having links without an underline feels like it goes against the basics of accessibility as you're relying on link states and styling to confirm a link.

Comment (hdfeddanjackson) [reply components/atoms/link/link.scss:7]: (I should add, I know this is not a comment on your code).

File: components/atoms/link/link.scss (hunk 1 above)
Comment (hdfedscottquested) [reply components/atoms/link/link.scss]: Done

File: components/atoms/link/link.scss (hunk 2 above)
Comment (hdfedscottquested) [reply components/atoms/link/link.scss]: Done

File: components/atoms/link/link.scss (hunk 3 above)
Comment (hdfedscottquested) [reply components/atoms/link/link.scss:7]: They do have an underline, it's just being added with the `:before` element. The standard underline is too close to the lettering and there is no way to move it.

File: components/atoms/link/link.scss [hunk 4]
```diff
@@ -1,46 +1,77 @@
...
+				&:after {
+					width: 100%;
+					background-color: $color-brand-blue-mid-light;
+				}
+			}
+
+			&:active,
+			&:focus {
+
+				&:before, &:after {
```
Comment (hdfeddanjackson) [inline components/atoms/link/link.scss:70]: I think they should be moved onto a new line:

```scss
&:before, 
&:after {
```

Comment (hdfedscottquested) [reply components/atoms/link/link.scss]: Done

**Review by hdfeddanjackson — APPROVED** [review]
Approved following changes.

**Pull request merged**